tools/linux_vsts.ps1

```diff
@@ -0,0 +1,300 @@
+$repoRoot = Join-Path $PSScriptRoot '..'
+Import-Module (Join-Path $repoRoot 'build.psm1') -Scope Global
+$isPR = $env:BUILD_REASON -eq "PullRequest"
+$commitMessage = git log --format=%B -n 1 $env:BUILD_SOURCEVERSION
```

Comment:
`$env:BUILD_SOURCEVERSIONMESSAGE` would be an easier way of retrieving the commit message.